src/GeositeFramework/js/Plugin.js

```diff
@@ -409,17 +417,28 @@ require(['use!Geosite',
                     selected: model.selected || model.get('active'),
                     displayed: model.selected,
                     fullName: model.get('pluginObject').fullName
-                }));
+                })),
+                sidebarNavSizedForTablet = !!$('.nav-apps-narrow'),
+                pluginContentHidden = $('.sidebar').filter(function() {
```

Comment:
I _believe_ this is referring to all plugins, not just "this" plugin.  If so you, you can use the same model state from the previous comment, but you'll access the collection with `view.model.collection`.